lib/eslint/eslint.js

```diff
@@ -1299,7 +1299,7 @@ class ESLint {
 	 * @returns {Promise<string|undefined>} The path to the config file being used or
 	 *      `undefined` if no config file is being used.
 	 */
-	findConfigFile(filePath) {
+	async findConfigFile(filePath) {
```

Comment:
Out of curiosity: Is it necessary to use async? (`return this.#configLoader.findConfigFileForPath()` already returns a promise)

Reply:
I think it's good for consistency that the method can never throw synchronously.